auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/AuthorizationClient.java

```diff
@@ -48,6 +48,24 @@ public void checkRight(String requestId, String userCrn, String right, String re
         );
     }
 
+    public boolean hasRight(String requestId, String userCrn, String right, String resource) {
```

Comment:
So the interface is hasRights, right? maybe mirror the actual interface?

Reply:
I agree with Gilad, it is worth to use list here in the same way as interface does